src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po

```diff
@@ -3808,6 +3824,9 @@ msgstr ""
 msgid "Receives notifications for internal notes on reservations."
 msgstr "Reçoit des notifications pour les notes internes sur les réservations."
 
+msgid "Receives notifications for rejected reservations."
+msgstr ""
```

Comment:
This was missing from French and Italian translations

Reply:
It's because the branch is based on `ogc-1068-interne-notizen`, not master. I did this preemptive merge because I expected this would avoid conflicts due to overlapping modifications. 
In hindsight that was clearly not the best idea. I will cherry-pick the changes and merge manually